### ISSUE TITLE
Issue with placing footprints in a circle

### ISSUE THREAD

**Issue by marc-cr1810** [MitjaNemec/PlaceFootprints#24]
No matter what the radius is set to, the plugin will ignore it and just use 0.

I am using KiCad v8.0

**Comment by MitjaNemec** [comment]
Hmmm, on what OS are you? What is your locale environments (OS language settings, KiCad language settings, units used)?

**Comment by huynguyen1989** [comment]
I think he mentioned about the parameter 
`Rotate every n-th footprint = 0` Will cause the error

<img width="326" alt="Screenshot 2024-08-31 at 00 48 37" src="https://github.com/user-attachments/assets/1559e090-323f-465a-a30a-2e9020b798d8">

**Comment by marc-cr1810** [comment]
> Hmmm, on what OS are you? What is your locale environments (OS language settings, KiCad language settings, units used)?

Just Windows 11, English for both and metric mm/cm

**Comment by marc-cr1810** [comment]
> I think he mentioned about the parameter 
> `Rotate every n-th footprint = 0` Will cause the error
> 
> <img width="326" alt="Screenshot 2024-08-31 at 00 48 37" src="https://github.com/user-attachments/assets/1559e090-323f-465a-a30a-2e9020b798d8">
> 

No, I did rotate every 15, and it did rotate them, but the positions were all dead centre and not placed at a distance from the centre

**Comment by huynguyen1989** [comment]
@marc-cr1810  Understood, 
I'm using MacOs and everything works so well. 
So, my suggestion is you should double check version of python on your OS with kicad's python or package  version requirements.

**Comment by ciriousjoker** [comment]
Fails on mac os for me:

[place_footprints.log](https://github.com/user-attachments/files/19575978/place_footprints.log)

Trying to lay out 28 leds in a circle
